.github/workflows/release.yml

```diff
@@ -16,8 +13,7 @@ jobs:
         uses: actions/checkout@v2
         with:
           fetch-depth: 0
-          ref: ${{ github.event.inputs.branch }}
-          # We need a personal access token to be able to push to a protected branch
+          ref: main  # Hardcoded to main branch
```

Comment:
Prior it was possible to push to maven central from any branch which can lead to accidental pushes to maven central